java/src/jmri/jmrix/acela/AcelaNode.java

```diff
@@ -135,22 +135,25 @@
     public static final String outputLEN0 = "0";        // used to dump/restore config file.
     public static final String outputNO = "N0";         // used to dump/restore config file.
     protected int startingOutputAddress = -1;           // used to aid linear address search
-    protected int endingOutputAddress = -1;           // used to aid linear address search
+    protected int endingOutputAddress = -1;             // used to aid linear address search
     protected int startingSensorAddress = -1;           // used to aid linear address search
-    protected int endingSensorAddress = -1;           // used to aid linear address search
+    protected int endingSensorAddress = -1;             // used to aid linear address search
 
     /**
-     * Assumes a node address of 0, and a node type of NO_CARD If this
+     * Assumes a node address of 0, and a node type of NO_CARD. If this
      * constructor is used, actual node address must be set using
-     * setNodeAddress, and actual node type using 'setNodeType'
+     * 'setNodeAddress()', and actual node type using 'setNodeType()'
```

Comment:
Please use links (`{@link method()}`) to the methods here instead of relying on single quotes.

Reply:
Reviewed all Javadoc, also in AbstracNode. Prefer to inherit when identical